src/core/portablemc/__init__.py

```diff
@@ -775,6 +790,22 @@ def cleanup():
                             with open(path.join(bin_dir, native_name), "wb") as native_file:
                                 shutil.copyfileobj(native_zip_file, native_file)
 
+        for bin_file in self.bin_files:
+            # Resolve a potential symlink and check if the binary exists before linking.
+            bin_file = path.realpath(bin_file)
+            if path.isfile(bin_file):
```

Comment:
Hm not sure how I feel about this, you can still give `--include-bin` a file that doesn't exist and it'll be silently ignored with no warning message or anything.

Reply:
You're right, I didn't know how to properly implement a feedback... Maybe a validation before starting the game, but I prefere an API validation. Maybe raising an error?

Reply:
Raising an error would make sense in my eyes [and is what I suggested before](https://github.com/mindstorm38/portablemc/pull/112#discussion_r1008039070) :)

Reply:
I missed it!